packages/accounts-base/server_main.js

```diff
@@ -4,9 +4,9 @@ import { AccountsServer } from "./accounts_server.js";
  * @namespace Accounts
  * @summary The namespace for all server-side accounts-related methods.
  */
-Accounts = new AccountsServer(Meteor.server, Meteor.settings.packages?.accounts || {});
+Accounts = new AccountsServer(Meteor.server, Meteor.settings.packages?.['accounts-base'] || {});
```

Comment:
So, there isn't an `accounts` option that can be provided here? :thinking:

If someone already provides `accounts` as a setting, wouldn't this break their code?

Reply:
There was no documented `accounts`, it was actually `accounts-base`, but I'm fine with using both.